CHANGES.md

```diff
@@ -3,22 +3,29 @@
 
 * [#167](https://github.com/serokell/universum/issues/164):
   'identity' has been deprecated.
+  _Migration guide:_ use 'Prelude.id' instead.
```

Comment:
I'd just say `id`, because it's exported from `Universum`. `Prelude.id` makes one think it's not in `Universum`.